### ISSUE TITLE
Add stats for tracking universal links

### ISSUE THREAD

**Issue by kwonye** [wordpress-mobile/WordPress-Android#8058]
The Reader currently has tracking in the following way:

```
DEEP_LINKED,
DEEP_LINKED_FALLBACK,
DEEP_LINK_NOT_DEFAULT_HANDLER,
```

and in each of these they include the URI the app was started with:

```
http[s]://wordpress.com/read/blogs/{blogId}/posts/{postId}
http[s]://wordpress.com/read/feeds/{feedId}/posts/{feedItemId}
http[s]://{username}.wordpress.com/{year}/{month}/{day}/{postSlug}
```

We do not mess with app opening at all.

@frosty do you think this is a viable solution for iOS?

**Comment by kwonye** [comment]
> Just to clarify – this event will be posted in addition to the normal app opening event, correct?

Yup!

> One 'issue': I just looked at the implementation of those tracks events, and it looks like the property keys intent_action and intent_data, which don't really make sense on the iOS side of things.

Yea, intent data can be android specific. The URL from which we open it from will be useful however, correct?

**Comment by frosty** [comment]
Yes, I guess so – we'd want to track which section we're opening as the result of a deep link. Do I need to use the same `intent_action` name on iOS? Does it help if it's the same across platforms?

**Comment by kwonye** [comment]
> Yes, I guess so – we'd want to track which section we're opening as the result of a deep link. Do I need to use the same intent_action name on iOS? Does it help if it's the same across platforms?

No that's just a property that can be android specific. Let's not add it to iOS if it doesn't make sense.
